### PR TITLE
fixed the language picker

### PR DESCRIPTION
**Pull request by sahilit**
fix the bug of LanguageScreen

### CODE REVIEW

**Comment by sahilit** [comment]
Also updated Expo packages to remove warnings

File: src/screens/ProfileScreen/LanguageScreen.js [hunk 1]
```diff
@@ -82,14 +82,53 @@ const styles = StyleSheet.create({
     color: '#F55963',
   },
   wheelPicker: {
+    // backgroundColor: '#ccc',
```
Comment (narender2031) [inline src/screens/ProfileScreen/LanguageScreen.js]: Please remove the commented Styles

File: src/screens/ProfileScreen/LanguageScreen.js [hunk 2]
```diff
@@ -82,14 +82,53 @@ const styles = StyleSheet.create({
...
+    height: 270,
     width: '100%',
-    flex: 1,
+    // flex: 1,
```
Comment (narender2031) [inline src/screens/ProfileScreen/LanguageScreen.js]: Please remove the commented Styles

File: src/screens/ProfileScreen/LanguageScreen.js [hunk 3]
```diff
@@ -82,14 +82,53 @@ const styles = StyleSheet.create({
...
+  },
 });
 
+const Bubble = props => {
+  const { children, selected } = props;
+  return (
+    <View
+      style={[
+        styles.itemStyleVertical,
+        selected ? styles.itemSelectedStyleVertical_ : styles.itemSelectedStyleVertical,
```
Comment (narender2031) [inline src/screens/ProfileScreen/LanguageScreen.js]: why you need different styles for this

File: src/screens/ProfileScreen/LanguageScreen.js [hunk 4]
```diff
@@ -82,14 +82,53 @@ const styles = StyleSheet.create({
...
+      >
+        {children}
+      </Text>
+    </View>
+  );
+};
+
 const LanguageScreen = () => {
-  const [selectedLanguageIndex, setSelectedLanguageIndex] = React.useState(0);
+  const [selectedLanguageIndex, setSelectedLanguageIndex] = React.useState(2);
```
Comment (narender2031) [inline src/screens/ProfileScreen/LanguageScreen.js]: Please set this to `0`

**Review by narender2031 — CHANGES_REQUESTED** [review]

**Review by sarthology — COMMENTED** [review]
Do changes narendra asked.

File: src/screens/ProfileScreen/LanguageScreen.js [hunk 5]
```diff
@@ -82,14 +82,53 @@ const styles = StyleSheet.create({
...
+    borderColor: '#fff',
+  },
+
+  itemSelectedStyleVertical_: {
+    backgroundColor: '#F55963',
+  },
 });
 
+const Bubble = props => {
+  const { children, selected } = props;
```
Comment (jamesgeorge007) [inline src/screens/ProfileScreen/LanguageScreen.js]:
```suggestion
const Bubble = ({children, selected}) => {
```

Comment (jamesgeorge007) [reply src/screens/ProfileScreen/LanguageScreen.js]: Avoid the extra step.

**Review by jamesgeorge007 — DISMISSED** [review]

File: src/screens/ProfileScreen/LanguageScreen.js (hunk 4 above)
Comment (sahilit) [reply src/screens/ProfileScreen/LanguageScreen.js]: Was Setting it to 2 so that the English would be default language

**Review by narender2031 — APPROVED** [review]

**Review by sarthology — APPROVED** [review]

**Pull request merged**